lua/nvim-treesitter/parsers.lua

```diff
@@ -1180,8 +1114,8 @@ list.query = {
     url = "https://github.com/nvim-treesitter/tree-sitter-query",
     files = { "src/parser.c" },
   },
-  readme_name = "Tree-sitter query language",
   maintainers = { "@steelsojka" },
+  readme_name = "Tree-Sitter query language",
```

Comment:
I'm late but it's actually capitalised as "Tree-sitter".

Reply:
yeah, noticed that on the update README PR as well